preupg/ui/report/tests.py

```diff
@@ -82,7 +82,7 @@ def test_stringify_children(self):
 #         tarball_path = a.scan_system()
 #
 #         xml_path, html_path = extract_tarball(tarball_path, self.temp_dir)
-#         self.assertTrue(len(parse_report(xml_path)) > 0)
```

Comment:
Does this test suite have any purpose here when it's commented or can it be removed?

Reply:
Those tests were run very long time ago and I'm not even sure how to run them. This should probably be reported as a new issue to be resolved.